### PR TITLE
Selenium: add selenium test to check the workspace details of workspace with single machine

### PR DESCRIPTION
**Pull request by SkorikSergey**
### What does this PR do?
This PR create a selenium test for testing the workspace details of a workspace with single machine.

The existed **WorkspaceDetailsTest** selenium test splitted  to **WorkspaceDetailsSingleMachineTest** and **WorkspaceDetailsComposeTest** tests.
In **WorkspaceDetailsComposeTest** will be tested the **Machines** and the **Env Variables** tabs in workspace created from the Java-MySql stack.
selection_093

In **WorkspaceDetailsSingleMachineTest** will be tested the **Overview**, the **Projects**, the **Installers** and the **Servers** tabs.
selection_094

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/7919

### CODE REVIEW

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/Consoles.java [hunk 1]
```diff
@@ -177,6 +179,20 @@ public void waitReferenceIsNotPresent(String referenceId) {
     redrawDriverWait.until(invisibilityOfElementLocated(By.id(referenceId)));
   }
 
+  public Boolean checkThatServerExists(String serverName) {
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/Consoles.java]: Do we really need `Boolean` here? May primitive type `boolean` be enough?

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/Consoles.java [hunk 2]
```diff
@@ -177,6 +179,20 @@ public void waitReferenceIsNotPresent(String referenceId) {
...
+    List<WebElement> webElements =
+        loadPageDriverWait.until(
+            visibilityOfAllElementsLocatedBy(
+                By.xpath("//div[contains(@id, 'runtime-info-reference-')]")));
+    for (WebElement we : webElements) {
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/Consoles.java]: Empty line above the `for` can improve readability of code.

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/WorkspaceDetailsComposeTest.java [hunk 3]
```diff
@@ -242,47 +142,22 @@ public void workingWithMachines() {
...
-    installers.put("JSON language server", false);
-    installers.put("PHP language server", false);
-    installers.put("Python language server", false);
-    installers.put("Simple Test language server", false);
-    installers.put("SSH", false);
-    installers.put("Terminal", true);
-    installers.put("TypeScript language server", false);
-    installers.put("Yaml language server", false);
-
+  private void createMap() {
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/WorkspaceDetailsComposeTest.java]: You can replace the method by one command:
```javascript
  private static final Map<String, String> EXPECTED_VARIABLES = ImmutableMap.of("MYSQL_DATABASE", "petclinic",
      "MYSQL_PASSWORD", "password",
      "MYSQL_ROOT_PASSWORD", "password",
      "MYSQL_USER", "petclinic");
```

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/WorkspaceDetailsSingleMachineTest.java [hunk 4]
```diff
@@ -0,0 +1,267 @@
...
+  @Inject private WorkspaceOverview workspaceOverview;
+  @Inject private MachineTerminal terminal;
+  @Inject private TestWorkspace testWorkspace;
+  @Inject private TestProjectServiceClient testProjectServiceClient;
+  @Inject private Consoles consoles;
+
+  @BeforeClass
+  public void setUp() throws Exception {
+    workspaceName = testWorkspace.getName();
+    createMap();
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/WorkspaceDetailsSingleMachineTest.java]: this method could by replaced by the command:
```javascript
  private static final Map<String, Boolean> EXPECTED_INSTALLERS = ImmutableMap.<String, Boolean>builder()
      .put("C# language server", false)
      .put("Exec", false)
      .put("File sync", false)
      .put("Git credentials", false)
      .put("JSON language server", false)
      .put("PHP language server", false)
      .put("Python language server", false)
      .put("Simple Test language server", false)
      .put("SSH", false)
      .put("Terminal", true)
      .put("TypeScript language server", false)
      .put("Yaml language server", false)
      .build();
```

**Comment by SkorikSergey** [comment]
ci-build

**Review by tolusha — APPROVED** [review]

**Review by dmytro-ndp — APPROVED** [review]

**Pull request merged**